README.md

```diff
@@ -72,42 +80,57 @@ Here's a schematic overview of our main differential expression experiment:
-microarray data and “untransformed” RSEM count data (preprocessed using the 
-`limma::voom` function). 
-2. RNA-seq’d samples were ‘titrated’ into the data set, 
-10% at a time (0-100%) resulting in eleven experimental sets for each n
-ormalization method. 
-3. Differentially expressed genes (DEGs) were identified using 
-the `limma` package. We compared the Her2 and LumA subtypes as well as Basal
-v. all other samples. 
-4. Lists of experimental DEGs were compared to standard gene 
-sets using Jaccard similarity. 
+1. All matched samples were considered when building the platform-specific “silver standards.”
+These standards are the genes that were differentially expressed at a specified False Discovery Rate (FDR) using data sets comprised entirely of one platform and processed in a standard way: log2-transformed 
+microarray data and “untransformed” RNA-seq data. 
+2. RNA-seq samples were "titrated" into the data set, 10% at a time (0-100%), replacing their matched array samples, resulting in eleven experimental sets for each normalization method. 
+3. Differentially expressed genes (DEGs) were identified usingthe `limma` package.
+For BRCA, we compared the Her2 and LumA subtypes as well as Basal v. all other subtypes. 
+For GBM, we compared the Classical and Mesenchymal subtypes as well as Proneural v. all other subtypes.
+4. Lists of experimental DEGs were compared to standard genesets using Jaccard similarity and Spearman rank correlation. 
+
+In the "small n" experiment, between 3 and 50 samples were selected from each subtype for DEG comparison.
 
 ```
 # Note: This requires the data to be processed to include matched samples only, 
 # and split into training and test sets (0-expression_data_overlap_and_split.R)
 
 # To run the differential expression pipeline, run in top directory:
-sh run_differential_expression_experiments.sh
+bash run_differential_expression_experiments.sh [cancer type] [subtype vs others] [subtype vs subtype] [subtype vs subtype small] [n cores]
 ```
 
+where 
+
+- `[cancer type]` is `BRCA` or `GBM`
+- `[subtype vs others]` is the subtype to be compared against all other subtypes
+- `[subtype vs subtype]` are the two subtypes to be compared (comma-separated, e.g. Her2,LumA)
+- `[subtype vs subtype small]` are the two subtypes to be compared at small sample sizes (comma-separated, e.g. Her2,LumA)
+- `[n cores]` is the number of cores you want to run in parallel
+
 ## Requirements
 
-This analysis was performed in R. It requires R & Bioconductor packages 
-detailed in `check_installs.R` to be installed.
+We recommend using the docker image `envest/rnaseq_titration_results:R-4.1.2` to handle package and dependency installation.
+See `docker/R-4.1.2/Dockerfile` for more information.
 
-One github package (`TDM`) is required. To install, run:
+Our analysis (v2.0) was run using 7 cores on an AWS instance with 16 cores, 128 GB memory, and an allocated 1 TB of space.
```

Comment:
How do you anticipate folks will interact with the container? Might want to add something like the `run` command here: https://github.com/AlexsLemonade/refinebio-examples/blob/staging/CONTRIBUTING.md#setting-up-the-docker-container

Reply:
I would also consider putting the Docker documentation under "Recreate manuscript results" with its own subheader.